roles/forkliftcontroller/defaults/main.yml

```diff
@@ -22,6 +22,7 @@ validation_image_fqin: "{{ lookup( 'env', 'VALIDATION_IMAGE') }}"
 validation_service_name: "{{ app_name }}-validation"
 validation_deployment_name: "{{ validation_service_name }}"
 validation_container_name: "{{ app_name }}-validation"
+validation_policy_agent_search_interval: "10"
```

Comment:
I know I requested 10 seconds but **60** seconds seems more reasonable.

Reply:
Increased to 120 (2 minutes) as requested